### PR TITLE
Update ITEM.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
5485 - 체이퍼러 = Chaferer? Not sure about the Korean here.

5470~

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5467,25 +5467,25 @@ ITEM_20150714_005466	Materials needed to create an explosive shot
...
-ITEM_20150714_005488	
+ITEM_20150714_005477	Perfect Trap
+ITEM_20150714_005478	Evoniphon's trap. It's in perfect working condition.
+ITEM_20150714_005479	Scripture Sheet
+ITEM_20150714_005480	Monk Wiley's treasured sheet of scripture. Looks like they were scattered during the uproar.
+ITEM_20150714_005481	Tempas Mushroom
+ITEM_20150714_005482	Mushrooms commonly found in the Thorn Forest. It seems they must be cooked to be edible.
+ITEM_20150714_005483	Rocktor's Stony Leg
+ITEM_20150714_005484	A piece of a Roctor's leg. Can be used in weapons due to its shape and hardness.
+ITEM_20150714_005485	Poisonous Chaferer Thorn
```
Comment (Sunaries) [inline ITEM.tsv]: Chaferer -> Chafperor

File: ITEM.tsv [hunk 2]
```diff
@@ -5467,25 +5467,25 @@ ITEM_20150714_005466	Materials needed to create an explosive shot
...
+ITEM_20150714_005486	A thorn found on the Chaferer's forearm. Venom oozes from it.
```
Comment (Sunaries) [inline ITEM.tsv]: See note above.

File: ITEM.tsv [hunk 3]
```diff
@@ -5467,25 +5467,25 @@ ITEM_20150714_005466	Materials needed to create an explosive shot
...
-ITEM_20150714_005487	
-ITEM_20150714_005488	
+ITEM_20150714_005477	Perfect Trap
+ITEM_20150714_005478	Evoniphon's trap. It's in perfect working condition.
+ITEM_20150714_005479	Scripture Sheet
+ITEM_20150714_005480	Monk Wiley's treasured sheet of scripture. Looks like they were scattered during the uproar.
+ITEM_20150714_005481	Tempas Mushroom
+ITEM_20150714_005482	Mushrooms commonly found in the Thorn Forest. It seems they must be cooked to be edible.
+ITEM_20150714_005483	Rocktor's Stony Leg
+ITEM_20150714_005484	A piece of a Roctor's leg. Can be used in weapons due to its shape and hardness.
```
Comment (Sunaries) [inline ITEM.tsv]: Roctor -> Rocktor

**Comment by imcgames** [comment]
Thank you! :smile:

**Pull request merged**